packages/fx-core/src/plugins/resource/sql/plugin.ts

```diff
@@ -446,4 +493,20 @@ export class SqlPluginImpl {
       this.config.sqlServer = this.config.sqlEndpoint.split(".")[0];
     }
   }
+
+  private loadDatabases(ctx: PluginContext) {
+    ctx.config.forEach((v: string, k: string) => {
+      if (k.startsWith(Constants.databaseName)) {
+        this.config.databases.push(v);
+      }
+    });
+  }
+
+  private removeDatabases(ctx: PluginContext) {
+    ctx.config.forEach((v: string, k: string) => {
+      if (k.startsWith(Constants.databaseName) && k !== Constants.databaseName) {
+        ctx.config.delete(k);
```

Comment:
can we merge loadDatabases and removeDatabases into one function?

Reply:
Actually, loadDatabases is used in both pre-provision and post-provision.
removeDatabases is used in pre-provision stage before loadDatabases. It is used to remove possible outdated database name. It is an additional action.